test/e2e_flags.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This file contains logic to encapsulate flags which are needed to specify
+// what cluster, etc. to use for e2e tests.
+
+package test
+
+const (
+	// ServingOperatorNamespace is the default namespace for serving operator e2e tests
+	ServingOperatorNamespace = "knative-serving"
+	// ServingOperatorNamespace is the default operator name for serving operator e2e tests
+	ServingOperatorName = "knative-serving"
```

Comment:
should we name the operator different than the namespace?

Reply:
@k4leung4 so far, no, because they are pre-installed(knative-serving/knative-serving) after we install the operator.
Will work on that feature on #104